README.rst

```diff
@@ -310,6 +310,34 @@ You can install requirements in development stage by:
     $ pip-sync requirements.txt dev-requirements.txt
 
 
+Version control integration
+---------------------------
+
+You might use ``pip-compile`` as a hook for the `pre-commit <https://github.com/pre-commit/pre-commit>`_.
+See `pre-commit docs <https://pre-commit.com/>`_ for instructions.
+Sample ``.pre-commit-config.yaml``:
+
+.. code-block:: yaml
+
+    repos:
+      - repo: https://github.com/jazzband/pip-tools
+        rev: 5.0.0
```

Comment:
Why 5.0.0?

Reply:
Never mind @atugushev I just saw this:
https://github.com/jazzband/pip-tools/milestone/32

Looks good to me then

Reply:
👍